plugin/distribution/pull.go

```diff
@@ -143,8 +143,7 @@ func Pull(name string, rs registry.Service, metaheader http.Header, authConfig *
 		logrus.Debugf("pull.go: error in json.Unmarshal(): %v", err)
 		return nil, err
 	}
-	if m.Config.MediaType != MediaTypeConfig &&
-		m.Config.MediaType != "application/vnd.docker.plugin.image.v0+json" {
+	if m.Config.MediaType != MediaTypeConfig {
```

Comment:
was `application/vnd.docker.plugin.image.v0+json` intended as a placeholder for `application/vnd.docker.plugin.v1+json`? What happens when `application/vnd.docker.plugin.v1+json` is encountered?